src/utils/awsAccess.ts

```diff
@@ -0,0 +1,96 @@
+import jwt from "@tsndr/cloudflare-worker-jwt";
+import { v4 as uuidv4 } from "uuid";
+import { env } from "../typeDefinitions/default.types";
+import config from "../../config/config";
+import { discordTextResponse } from "./discordResponse";
+import { DISCORD_BASE_URL, AWS_IAM_SIGNIN_URL } from "../constants/urls";
+
+export async function processAWSAccessRequest(
+  discordUserId: string,
+  awsGroupId: string,
+  env: env,
+  TraceId: string,
+  channelId: number
+) {
+  const authToken = await jwt.sign(
+    { name: "Cloudflare Worker", exp: Math.floor(Date.now() / 1000) + 2 },
```

Comment:
What is the name here?

Reply:
in the website backend, we check if the auth token has the name "cloudfare workers" to validate the token coming from right source.

Reply:
how is a string helping in validating the right source?

Reply:
So we create a token in the discord slash commands with name present in payload, this is signed by using the private key and in website backend we validate if the same text is being received post validating the token.
@prakashchoudhary07

Reply:
I think we need to discuss on this one

Reply:
sure @prakashchoudhary07